src/components/pages/devices/flyouts/deviceDetails/deviceDetails.scss

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/mixins';
+@import 'src/styles/themes';
+
+.device-details-container {
+  display: flex;
+  flex-flow: column nowrap;
+  @include rem-font-size(14px);
+
+  .flyout-section {
+    .section-content {
+      @include rem-fallback(padding, 0px, 12px, 0px, 12px);
+
+      .basic-row {  @include rem-fallback(padding-top, 16px); }
+      .basic-cell { @include rem-fallback(line-height, 17px); }
+    }
+  }
+
+  .device-details-header {
+    @include rem-fallback(padding, 12px, 12px, 12px, 12px);
+
+    .device-name { @include rem-fallback(font-size, 18px); }
+    .device-simulated { @include rem-fallback(font-size, 10px); }
+    .device-connected { @include rem-fallback(font-size, 14px); }
+
+    .device-icon svg {
+      @include rem-fallback(height, 72px);
+      @include rem-fallback(margin-right, 24px);
+    }
+  }
+
+  @include themify($themes) {
```

Comment:
I aim to have only one call to themify per sass file. It is easier to modify if all the theming vars are in one place and one location (the end of the file)